cpacker/MemGPT
# feat: Feature/global logging

**Please describe the purpose of this pull request.**
Reconfigure global logging to use Constants instead of config

**How to test**
To test you can run ```memgpt configure``` it should create a directory .memgpt/logs/MemGPT.log 

This only exists in config.py with only one debug for save. 

**Have you tested this PR?**
This is the first stage.  I will be creating a pytest in a commit soon. 

**Related issues or PRs**
https://github.com/cpacker/MemGPT/issues/319

**Is your PR over 500 lines of code?**
No.
